aries_cloudagent/resolver/default/indy.py

```diff
@@ -37,6 +37,10 @@ def supported_methods(self) -> Sequence[str]:
         """Return supported methods of Indy DID Resolver."""
         return ["sov"]
 
+    async def supports(self, profile: Profile, did: Union[str, DID]) -> bool:
```

Comment:
The inherited class already has this method with the same behavior, hasn't it?

Reply:
please comment on pr #84

Reply:
I romoved this..